spec/unit/installer/file_references_installer_spec.rb

```diff
@@ -42,7 +42,10 @@ module Pod
         file_ref.path.should == 'libBananalib.a'
       end
 
-      it 'adds the files references of the resources the Pods project' do
+      # TODO: Remove this as PR #3263 does not add the resources to the Pods project anymore
```

Comment:
Will remove. Should be replaced by a spec checking that `it 'adds the files references of the resources to the User's project'` instead, which should be somewhere around `user_project_integrator_spec.rb` or something.